docs/tests/node-e2e-ci.md

```diff
@@ -0,0 +1,167 @@
+# Running Kubernetes node e2e Conformance tests on RHEL
+
+Currently, we have two periodic Jenkins jobs running Kubernetes node e2e Conformance tests on RHEL.
```

Comment:
perhaps clarify who "we" are in this situation? *I* can gather this, but other readers would probably assume that test-infra-maintainers were running this.